buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -677,8 +677,8 @@ class BeamModulePlugin implements Plugin<Project> {
         activemq_junit                              : "org.apache.activemq.tooling:activemq-junit:$activemq_version",
         activemq_kahadb_store                       : "org.apache.activemq:activemq-kahadb-store:$activemq_version",
         activemq_mqtt                               : "org.apache.activemq:activemq-mqtt:$activemq_version",
-        antlr                                       : "org.antlr:antlr4:4.7",
-        antlr_runtime                               : "org.antlr:antlr4-runtime:4.7",
+        antlr                                       : "org.antlr:antlr4:4.10",
+        antlr_runtime                               : "org.antlr:antlr4-runtime:4.10",
```

Comment:
antlr is only used by Beam SDK core and is shaded. We should be able to unpin antlr. Let me open a PR